### ISSUE TITLE
Missing docstring for TFTDRandomFourier. Check Documenter's build log for details.

### ISSUE THREAD

**Issue by Datseris** [JuliaDynamics/TimeseriesSurrogates.jl#105]
In our docs: 
![image](https://user-images.githubusercontent.com/19669089/150124001-875beb15-e110-471a-8166-49ce6f73ccc9.png)

@kahaaga I am assigning you because I think (but not sure), this was a method you contributed...?

**Comment by kahaaga** [comment]
Yep, I contributed this. An extra line snuck in that detached the doctoring from the function. I took the opportunity to improve the doctoring a bit too.
